tests/scr-support_iframe/OPS/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Scripting</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2021-12-10</dc:date>
+    <dc:description>Checks whether the reading system implements scripting in a container-constrained document (i.e., whether a simple inline script is executed at all).</dc:description>
+    <dc:identifier id="pub-id">scr-support_iframe</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Scripting support</dc:title>
```

Comment:
...in an iframe? (Applies to other <titles> in the same test too.)

Reply:
I have actually changed the text to 

> Checks whether the reading system implements container-constrained scripts (i.e., whether a simple inline script is executed at all in an iframe).

Per [content spec](https://www.w3.org/TR/epub-33/#sec-scripted-container-constrained) "container-constrained script" is the official statement. (Will do in all other tests.)